library/src/main/java/com/getstream/sdk/chat/rest/controller/RetrofitClient.java

```diff
@@ -112,4 +114,10 @@ public static Retrofit getAuthorizedCDNClient(CachedTokenProvider tokenProvider,
                 .addConverterFactory(GsonConverterFactory.create(GsonConverter.Gson()))
                 .build();
     }
+
+    private static HttpLoggingInterceptor setupLoggingInterceptor() {
+        loggingInterceptor = new HttpLoggingInterceptor();
+        loggingInterceptor.level(HttpLoggingInterceptor.Level.BODY);
```

Comment:
@tbarbugli do you remember if we disabled it deliberately some time ago or not?

Reply:
@elevenetc i can disable it if needed